src/views/RecIM/components/CreateActivityForm/CreateActivityForm.module.scss

```diff
@@ -1,12 +0,0 @@
-@import '../../../../vars';
-
-.form_title {
-  background-color: $primary-blue;
-  color: $neutral-white;
-  text-align: center;
-  grid-row: 1;
-
-  & :global(.MuiTypography-h6) {
-    font-size: 2em;
-  }
-}
```

Comment:
I do not mind you deleting the contents of this file however, I don't know if you should be deleting the scss of this index.